src/plugins/memdump/memdump.cpp

```diff
@@ -114,6 +114,40 @@
 
 #define DUMP_NAME_PLACEHOLDER "(not configured)"
 
+static void save_file_metadata(const drakvuf_trap_info_t* info,
+                               const char* file_path,
+                               size_t dump_size,
+                               addr_t dump_address,
+                               const char* method,
+                               const char* dump_reason,
+                               extras_t* extras)
+{
+    char* file = NULL;
+    if ( asprintf(&file, "%s.metadata", file_path) < 0 )
+        return;
+
+    FILE* fp = fopen(file, "w");
+    free(file);
+    if (!fp)
+        return;
+
+    fprintf(fp, "Method: \"%s\"\n", method);
```

Comment:
JSON maybe? Or something easy to parse?

Reply:
This very easy to parse with Python: each line stores "KEY: VALUE", with strings in quotes.